CMakeLists.txt

```diff
@@ -1159,10 +1159,22 @@ if(IREE_BUILD_COMPILER)
     ADD_TO iree-install-tools
   )
 
+  iree_add_install_target(
+    NAME iree-install-tools-compiler-extra
+    COMPONENT IREETools-CompilerExtra
+    ADD_TO iree-install-tools
+  )
+
+  iree_add_install_target(
```

Comment:
Can you add a todo here to move this to the IREETools-CompilerExtra instead of it being its own component. I can fix it later -- don't want to add more burden to you

Reply:
Fixed.